models/phenom_beta/config.json

```diff
@@ -0,0 +1,68 @@
+{
```

Comment:
why are there two of these config jsons?

Reply:
one for loading the last.pickle and one for loading the model.safetensors, huggingface expects the config.json to be in the folder